C/README.md

```diff
@@ -58,7 +58,7 @@
 - [Bellman Ford Algorithm](ds/bellman_ford.c)
 - [Implementation of Dijkstra's Algorithm](ds/Dijsktra.c)
 - [Circular Queue](ds/circularqueue.c)
-- [Post Order Traversal](ds/Postorder.c)
```

Comment:
Don't remove anything from the README. You just need to add the link of your program under appropriate section.

Reply:
This issue has not been resolved yet.